src/codelabs/async-await.md

```diff
@@ -44,13 +59,26 @@ The following example shows the wrong way to use an asynchronous function
 Before running this example, try to spot the issue -- what do you think the
 output will be?
 
-<iframe
-  src="{{site.dartpad-embed}}?id=5c8c7716b6b4284842f15fe079f61e47"
-  style="border: 1px solid lightgrey; margin-top: 10px; margin-bottom: 25px"
-  frameborder="no"
-  height="420"
-  width="100%" >
-</iframe>
+<?code-excerpt "async_await/bin/get_order_sync_bad.dart" remove="Fetching"?>
+```dart:run-dartpad:height-380px
+// This example shows how *not* to write asynchronous Dart code.
+
+String createOrderMessage() {
+  var order = getUserOrder();
+  return 'Your order is: $order';
+}
+
+Future<String> getUserOrder() =>
+    // Imagine that this function is more complex and slow
+    Future.delayed(
+      Duration(seconds: 2),
+      () => 'Large Latte',
+    );
+
+void main() {
```

Comment:
Using `void main()` here means that there's one more thing that changes in the asynchronous example later. (We intentionally left the return type off of `main()` to simplify the example... although we definitely considered other approaches.)

Reply:
Reverted; i.e., removed `void`.